### PR TITLE
Bux Fixes

### LINKED ISSUE #79: ''services'' option should be replaced with ''what we do''
''services'' option should be replaced with ''what we do'' on the upper menu

### PR DESCRIPTION
**Pull request by AndrewBestbier**
### Motivation

This is a general bug fixes PR. This closes #79 #78 #76  #75

### Test plan

View each issue and ensure it is fixed on all browsers accordingly.

### Pre-merge checklist

- [x] Unit tests
- [x] Reviews
  - [x] Code review
  - [ ] Design review
- [x] Manual testing
  - [x] Windows 7 IE 11
  - [x] Windows 10 Edge
  - [x] Mac OS El Capitan Safari
  - [x] Mac OS El Capitan Chrome
  - [x] Mac OS El Capitan Firefox
  - [x] iOS 9 Safari
  - [x] Android 6 Chrome
- [ ] Tester approved

### CODE REVIEW

File: site/components/footer/style.css [hunk 1]
```diff
@@ -41,7 +42,8 @@
 }
 
 .social {
-  composes: fontS from "../../css/typography/_fonts.css";
+  font-size: 24px;
```
Comment (AndrewBestbier) [inline site/components/footer/style.css]: Remove

File: site/layout/style.css [hunk 2]
```diff
@@ -4,6 +4,14 @@
   box-sizing: border-box;
 }
 
+input,
+textarea,
+button,
+select,
+a {
+  -webkit-tap-highlight-color: rgba(0,0,0,0);
```
Comment (asavin) [inline site/layout/style.css]: autoprefixer

**Comment by asavin** [comment]
Underline on blog slice is remaining after you click on it and go back

**Comment by AndreiDanielIonescu** [comment]
''what we do'' link in the footer directs the user to the ''about us'' page on the old site.

**Pull request merged**